src/promptflow-tools/tests/conftest.py

```diff
@@ -92,7 +97,8 @@ def skip_if_no_api_key(request, mocker):
         conn_name = request.node.get_closest_marker('skip_if_no_api_key').args[0]
         connection = request.getfixturevalue(conn_name)
         # if dummy placeholder key, skip.
```

Comment:
revert changes in conftest.py. Please add these changes when you add tests about your tool.